tests/test_generate_readme

```diff
@@ -0,0 +1,15 @@
+"""This module tests the generate_readme module."""
```

Comment:
If you decide to remove `generate_readme`, make sure to remove this file as well.